src/pages/PrimeCalculator/RewardDetails/index.tsx

```diff
@@ -1,3 +1,98 @@
-import { Card } from 'components';
+import { Card, Delimiter, Icon, TokenIcon, Tooltip } from 'components';
+import { useTranslation } from 'packages/translations';
+import { Token } from 'types';
 
-export const RewardDetails: React.FC = () => <Card>RewardDetails</Card>;
+interface RewardDetailsProps {
+  token: Token;
+  totalYearlyRewards: string;
+  userYearlyRewards: string;
+  userSuppliedTokens: string;
+  userBorrowedTokens: string;
+  primeSupplyApy: string;
+  primeBorrowApy: string;
+}
+
+export const RewardDetails: React.FC<RewardDetailsProps> = ({
+  primeBorrowApy,
+  primeSupplyApy,
+  token,
+  totalYearlyRewards,
+  userYearlyRewards,
+}: RewardDetailsProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card>
+      <h4 className="text-lg font-semibold">{t('primeCalculator.rewardDetails.title')}</h4>
+      <div className="mb-3 mt-6 flex flex-row justify-between">
```

Comment:
Since `flex-direction: row` is the default CSS rules applied to an element that's given `display: flex`, you don't need to apply it again here so you can remove the `flex-row` class (you can remove it in other places in this component too).